### ISSUE TITLE
Path to spanish docs contribution?

### ISSUE THREAD

**Issue by sebassdc** [pybee/batavia#670]
I'll like to help translating the batavia docs. I will copy the actual docs to translate them line to line. What folder structure would you recommend me to put the spanish doc? I mean, on what path.

**Comment by freakboy3742** [comment]
@sebassdc Good question! We haven't started looking at translations for individual projects - to date, our translation efforts have been focussed on the main website. 

A great way to contribute would be to sort out what exactly we need to do to translate Sphinx documentation. A good starting point will be [this document](http://www.sphinx-doc.org/en/stable/intl.html). You're in new territory here, so any recommendation you make will probably be accepted!

**Comment by sebassdc** [comment]
Do you like the sphinx way? I would recomend another service like gitbook, its all markdown wich is more easy for everyone. I can help with the migration and the internationalitation. What do you think @freakboy3742 ?

**Comment by freakboy3742** [comment]
Yes, we're going to keep Sphinx for docs. I don't agree markdown is better for long-form documentation - it lacks many features for multiple-page documents. It's also the standard for Python projects generally.

**Comment by sebassdc** [comment]
Ok, I will investigate sphinx Internationalization and help with that. But check the gitbook [link](https://toolchain.gitbook.com/languages.html) and how the multi-page stuff is managed. Its very **simple**. I mean markdown its standard on many areas. It would be easiest to have more people with markdown knowledge
